tests/markdown_table/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -11,6 +11,28 @@ exports[`align.md 1`] = `
 
 `;
 
+exports[`cjk.md 1`] = `
+| abc | def | ghi |
+| --- | --- | --- |
+| 第一欄 | 第二欄 | 第三欄 |
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+| abc    | def    | ghi    |
+| ------ | ------ | ------ |
+| 第一欄 | 第二欄 | 第三欄 |
+
+`;
+
+exports[`emoji.md 1`] = `
+| abc | def | ghi |
+| --- | --- | --- |
+| 👍👍👍 | 👍👍👍 | 👍👍👍 |
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+| abc    | def    | ghi    |
+| ------ | ------ | ------ |
+| 👍👍👍 | 👍👍👍 | 👍👍👍 |
```

Comment:
![image](https://user-images.githubusercontent.com/1297597/31575380-0385c886-b129-11e7-8322-7d93637aab30.png)

These don't line up in my diff. Is this intentional?

Reply:
It's based on the font, if you use [monospace-font](https://github.com/ikatyang/configs/tree/master/fonts) that supports CJK character then you'll see:

![image](https://user-images.githubusercontent.com/8341033/31575402-d3bdbdde-b0ab-11e7-8995-20ad7733cb3e.png)